CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# Contributing to Kubeflow Examples
+
+You want to contribute to Kubeflow examples? That's awesome! Please refer to the short guide below. 
+
+# Contributing Guide
+
+The [Kubeflow](https://github.com/kubeflow/kubeflow/blob/master/README.md) project is dedicated to making machine learning on Kubernetes simple, portable and scalable. We need your support in making
+this repo the destination for top models and examples, which show the power of Kubeflow. We have created an initial list of
+proposed manifests. Please feel free to self-assign these examples, by following a simple 3 step process:
+
+* Identify an **example** in table below and put your github id against it
+* Create a Github issue with the details of the **example** and self-assign
+* Send a PR to this repo with the actual work for the example
+
+We have assigned priorities to the items below. See priority guidance: 
+
+* **P0**: Very important, try to self-assign if there is a P0 available
+* **P1**: Important, try to self-assign if there is no P0 available
+* **P2**: Nice to have
+
+# Proposed Examples
```

Comment:
Can you add GitHub Issue summarization?
https://github.com/kubeflow/kubeflow/issues/157

Reply:
I tried to add a line for #157, but got a permission denied error. @puneith if you add me as a contributor to your fork, I'll push my commit.

Reply:
Michelle added you as contributor.

Reply:
Added a line for GH issue summarization.

Reply:
@texasmichelle can you add link to GH issue#